iModelJsNodeAddon/api_package/ts/src/NativeLibrary.ts

```diff
@@ -441,9 +441,22 @@ export declare namespace IModelJsNative {
 
   interface SchemaImportOptions {
     readonly schemaLockHeld?: boolean;
+    readonly schemaSyncDbUri?: string;
     readonly ecSchemaXmlContext?: ECSchemaXmlContext;
   }
 
+  interface LocalDbInfo {

```

Comment:
please call this SchemaLocalDbInfo or something
```suggestion
  interface SchemaLocalDbInfo {
```